src/python/foglamp/admin_api/login.py

```diff
@@ -65,19 +58,21 @@
 
 @authentication_required
 async def get_user(request):
-    '''An example method that responds with the currently logged in user's details in JSON format'''
+    """An example method that responds with the currently logged in user's details in JSON format"""
     return json_response({'user': str(request.user)})
```

Comment:
can we fix __repr__ template for User class? make template a json so that we get **{user:**` {id:1, name:"username", is_admin:0}`**}**